package.json

```diff
@@ -84,7 +84,7 @@
     "wait-on": "6.0.0"
   },
   "lint-staged": {
-    "*.{ts,tsx,js,jsx}": "npm run lint -- --fix",
-    "*.{ts,tsx,js,jsx,scss,css,md}": "npm run format"
+    "*.{ts,tsx,js,jsx}": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
```

Comment:
Not 100% sure about the Prettier line. ESLint does run Prettier via `eslint-config-prettier`, so this _should_ work.